configure.ac

```diff
@@ -263,6 +263,12 @@ AC_ARG_ENABLE(
 	[enable_async_push="no"]
 )
 
+AC_ARG_ENABLE(vlan-tagging,
+	[  --disable-vlan-tagging  Disable support for 802.1Q-based VLAN tagging],
```

Comment:
minor thing here: I think this line is not following the same style as the other help messages. I.e. no AS_HELP_STRING